app/src/main/java/com/example/bookclub/BookDetailsActivity.java

```diff
@@ -61,85 +65,66 @@ protected void onCreate(Bundle savedInstanceState) {
 
         queue = Volley.newRequestQueue(this);
 
-        mBookImg = findViewById(R.id.mBookImg);
-        mBookTitle = findViewById(R.id.mTitleID);
-        mBookAuthor = findViewById(R.id.mAuthorID);
-        mBookPublisher = findViewById(R.id.mPublishedBy);
-        mBookPages = findViewById(R.id.mPages);
+        mAddToLibrary = findViewById(R.id.tvAddToLibrary);
+        mBookImg = findViewById(R.id.ivBookImg);
+        mBookTitle = findViewById(R.id.tvTitleID);
+        mBookAuthor = findViewById(R.id.tvAuthorID);
+        mBookPublisher = findViewById(R.id.tvPublishedBy);
+        mBookPages = findViewById(R.id.tvPages);
 
-        shareButton = findViewById(R.id.mShareID);
-        viewButton = findViewById(R.id.mViewID);
+        mAddToLibrary.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
```

Comment:
We should move this code into its own function. Will clean up the `OnCreate` code a bit.